app/jobs/resolution_proofing_job.rb

```diff
@@ -48,7 +48,9 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
                           )
                         end
 
-    add_threatmetrix_result_to_callback_result(callback_log_data.result, threatmetrix_result)
+    if use_lexisnexis_ddp_threatmetrix_before_rdp_instant_verify?
+      add_threatmetrix_result_to_callback_result(callback_log_data.result, threatmetrix_result)
+    end
```

Comment:
with the old code, the underlying feature flag was a string, so the `false` value was `"false"` which is truthy... meaning I think we need to watch out for this in the deploy